source/eventHandler.py

```diff
@@ -413,8 +414,8 @@ def shouldAcceptEvent(eventName, windowHandle=None):
 
 	# #6713: Edge (and soon all UWP apps) will no longer have windows as descendants of the foreground window.
 	# However, it does look like they are always  equal to or descendants of the "active" window of the input thread. 
+	gi = winUser.getGUIThreadInfo(0)

```

Comment:
could this get a more descriptive name?

Reply:
I agree it could be more descriptive. However:
* that variable always existed and was only moved up out of the if condition
* There is risk in changing that variable as a lot of tricky tests would have to be performed unrelated to this change.
If a sighted person or someone who is well versed in an IDE that can easily handle variable renames wants to do this cleanup they can in a separate PR.